### ISSUE TITLE
pip uninstall decord removes numpy and pip!

### ISSUE THREAD

**Issue by azkalot1** [dmlc/decord#149]
```
 pip3 uninstall decord
 ```
 
 ```
 Would remove:
 ....
 /home/skolchenko/miniconda3/envs/clean_deeplearning/lib/python3.8/site-packages/pip/*
 /home/skolchenko/miniconda3/envs/clean_deeplearning/lib/python3.8/site-packages/numpy/*
 ```
 
 I am quite sure it should not remove numpy and pip :)

**Comment by sbebo** [comment]
Found the same issue I think with the latest couple of releases (0.5.1, 0.52).

**Comment by zhreshold** [comment]
Fixed in #160, I am pushing 0.6.0 to fix this wheel
